.env

```diff
@@ -1,7 +1,9 @@
 ## Set this to true if you're using a load balancer, or set it to false if you're using seperate IPs for each service.
+## If you're using monolithic (the default), leave this set to true
 USE_GENERIC_CACHE=true
 
-## IP Address that the lancache load balancer will be running on
+## IP Address that the lancache monolithic instance will run on
+## Specify multiple IPs, space separated, if your lancache host has them - this can improve cache priming performance for some services (e.g. Steam)
 LANCACHE_IP=10.0.39.1
```

Comment:
move this below the DNS_BIND_IP, consider setting it by default to be ${DNS_BIND_IP} (this expansion does work)

Reply:
I think this potentially makes things more confusing for first time users, not less. By explicitly listing the IPs it adds a slight amount of repetition, but it doesn't raise the question of whether that's a required thing.

I think we should expand the `README.md` to better explain which of these settings matter, and what they do.

Reply:
I've added more documentation into the `README.md` that helps to explain the settings.

Reply:
@Lepidopterist pointed out there was some confusion if you just read the .env as this isn't the IP the cache will "bind" to, but the ip's the dns will serve. The wording makes it seem 0.0.0.0 is possible to bind to "everything"